ush/sfc_climo_gen.sh

```diff
@@ -79,8 +79,8 @@ vegetation_greenness_method="bilinear"
 /
 EOF
 
-APRUN_SFC=${APRUN_SFC:-"aprun -j 1 -n 6 -N 6"}
-$APRUN_SFC $EXEC_DIR/sfc_climo_gen
+SRUN_SFC=${SRUN_SFC:-"srun -A gsd-fv3-dev -J sfc_climo -N 6"}
```

Comment:
I tried to create a C96 global grid on Hera.  It failed at this step.  The grid driver scripts pass in APRUN_SFC, not SRUN_SFC.  Why was this changed?

Reply:
(1) sfc_climo_gen.sh is reverted.
(2) this branch has merged to the develop top.
(3) add_lake = T works with global and regional for now. An error message will show up in the combination of add_lake = T with stretch or nested. 
(4) If needed, we could get add_lake = T to work with stretch or nested at a later time.
(5) meta data is updated.

Please let me know if you have any questions. Thanks,

Shan

Reply:
sfc_climo_gen.sh works now.  Thanks.